life_expectancy/country.py

```diff
@@ -0,0 +1,52 @@
+"""Enum file"""
```

Comment:
Does not provide information about its content
```suggestion
"""Information on Countries' Available at */partial/path/to/file"""
```

Reply:
Agreed, it is a more informative docstring